homeassistant/components/agent_dvr/camera.py

```diff
@@ -67,6 +67,9 @@ async def async_setup_entry(
 class AgentCamera(MjpegCamera):
     """Representation of an Agent Device Stream."""
 
+    _attr_should_poll = True
```

Comment:
This won't work because the `should_poll` property in `Entity` is overwritten in `Camera`.

It might be better to adjust all the internal integrations first. ("quality_scale": "internal")

Relevant classes:
https://github.com/home-assistant/core/blob/bdf247faaab79cefdbee5f7a24cb68d07576b4cc/homeassistant/helpers/entity.py#L240-L246

https://github.com/home-assistant/core/blob/bdf247faaab79cefdbee5f7a24cb68d07576b4cc/homeassistant/components/camera/__init__.py#L329-L332

https://github.com/home-assistant/core/blob/bdf247faaab79cefdbee5f7a24cb68d07576b4cc/homeassistant/components/mjpeg/camera.py#L87

Reply:
Yup, this needs other changes first. Not everything can use the new attributes yet.